cmake/FindLuaJIT.cmake

```diff
@@ -15,35 +15,36 @@
 #
 #######################
 
-# FindLuaJIT.cmake
+# Findluajit.cmake
 #
 # This will define the following variables
 #
-#     LuaJIT_FOUND
+#     luajit_FOUND
 #
 # and the following imported targets
 #
-#     LuaJIT::LuaJIT
+#     luajit::luajit
 #
 
-# LuaJIT exports their own config since LuaJIT-1.5.0, but it isn't
-# present in the OpenSUSE libLuaJIT-devel-1.7.1 package and maybe others.
```

Comment:
I do not know where I got this information - it appears to be completely wrong. Might as well remove it entirely.